source/_components/light.http_rgb.markdown

```diff
@@ -0,0 +1,29 @@
+---
+layout: page
+title: "http_rgb"
```

Comment:
Please stick to a name. So far there is `http_rgb`, `HTTP_RGB` and `HTTP_RGB!`. `http_rgb` is the platform name.

Reply:
Agreed - I've updated my pull request on home-assistant/home-assistant to reflect a consistent naming standard.